config/routes/admin.rb

```diff
@@ -41,8 +41,9 @@
 
       resources :tag_rules, only: [:destroy]
 
-      # TODO do we need to remove more routes
-      resources :vouchers, only: [:new, :create]
+      constraints FeatureToggleConstraint.new(:vouchers) do
+        resources :vouchers, only: [:new, :create]
+      end
```

Comment:
I don't think that we need to restrict routes. It's not that people are not allowed to use it. We just don't want to show it yet because it's experimental.